src/main/resources/io/jenkins/plugins/customizable_header/headers/LogoHeader/headerContent.jelly

```diff
@@ -1,9 +1,9 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:h="/lib/layout/header">
   <template id="custom-header-icons">
     <l:icon id="external-link" src="symbol-open-outline plugin-ionicons-api" class="icon-xs"/>
   </template>
-  <header id="header" class="page-header custom-header__page">
+  <header id="page-header" class="page-header custom-header__page">
```

Comment:
adjusted to match core as https://github.com/jenkinsci/jenkins/pull/7569 was going to use it, although I removed the hard dependency on it there in https://github.com/jenkinsci/jenkins/pull/7569/commits/6092031216b01211ca612a143ca826d709337739 there is at least one other reference to it